main/src/main/java/org/mobilitydata/gtfsvalidator/validator/UrlConsistencyValidator.java

```diff
@@ -115,9 +115,11 @@ private ListMultimap<String, GtfsRoute> routesByUrlMap(GtfsRouteTableContainer r
    */
   private ListMultimap<String, GtfsAgency> agenciesByUrlMap(GtfsAgencyTableContainer agencyTable) {
     ListMultimap<String, GtfsAgency> agenciesByUrl = ArrayListMultimap.create();
-    for (GtfsAgency agency : agencyTable.getEntities()) {
-      if (agency.hasAgencyUrl()) {
-        agenciesByUrl.put(Ascii.toLowerCase(agency.agencyUrl()), agency);
+    if (agencyTable != null) {
+      for (GtfsAgency agency : agencyTable.getEntities()) {
+        if (agency.hasAgencyUrl()) {
+          agenciesByUrl.put(Ascii.toLowerCase(agency.agencyUrl()), agency);
+        }
```

Comment:
[suggestion]: Have this code in its own PR with similar protecting conditionals added to this PR.

Reply:
Actually, the code is not needed once agency.txt is replaced by networks.txt in the GtfsNetworkSchema class. I'll remove them.